src/printer.js

```diff
@@ -441,7 +439,9 @@ function genericPrintNoParens(path, options, print, args) {
                 path,
                 print,
                 options,
-                args && (args.expandLastArg || args.expandFirstArg)
+                /* expandLast */ args &&
+                  (args.expandLastArg || args.expandFirstArg),
+                /* printTypeParams */ true
```

Comment:
There are some cases like methods where it makes sense for the type params to be in the pattern parent group.